### DISCUSSION TITLE
Pull request template

### DISCUSSION THREAD

**Discussion by shloka-gupta** [directus/directus#13521] [Feature Requests]
Add a [pull request](https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/) template to set some ground rules for what types of PRs are accepted.

**Comment by rijkvanzanten** [comment]
I think the example template from the article you linked could be a good starting point: https://github.com/embeddedartistry/templates/blob/master/oss_docs/PULL_REQUEST_TEMPLATE.md

**Comment by rijkvanzanten** [comment]
This was since added!
